IframeMediaPause/iframe_media_pausing.md

```diff
@@ -0,0 +1,189 @@
+
+## Goals
+
+Propose a mechanism to allow embedder documents to limitedly control embedded iframe media playback based on whether the embedded iframe is [rendered](https://html.spec.whatwg.org/multipage/rendering.html#being-rendered) or not:
+- When the iframe is not rendered, the embedder is able to pause the iframe media playback; and
+-  When the iframe becomes rendered again, the embedder is able to resume the iframe media playback.
+
+## Non-Goals
+It is not a goal of this proposal to allow embedders to arbitrarily control when to play, pause, stop, resume, etc, the media playback of a rendered iframe.
+
+## Use Cases
+There are scenarios where a website might want to just not render an iframe. For example:
+- A website, in response to an user action, might decide to temporarily not show an iframe that is playing media. However, since it is not possible to mute it, the only option is for the website to remove the iframe completely from the DOM and recreate it from scratch when it should be visible again. Since the embedded iframe can also load many resources, the iframe recreation operation might make the web page slow and spend resources unnecessarily.
+
+## Proposed solution: media-playback-while-not-rendered Permission Policy
+
+We could create new "media-playback-while-not-rendered" [Permission Policy](https://www.w3.org/TR/permissions-policy/) that would pause any media being played by iframes which are not currently [rendered](https://html.spec.whatwg.org/multipage/rendering.html#being-rendered). This would apply whenever the iframe’s `"display"` CSS property is set to `"none"`.
+
+This policy will have a default value of '*', meaning that all of the nested iframes are allowed to play media when not rendered. The example below show how this permission policy could be used to prevent all the nested iframes from playing media. By doing it this way, even other iframes embedded by "foo.media.com" shouldn’t be allowed to play media if not rendered.
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>iframe pausing</title>
+  </head>
+  <body>
+    <iframe src="https://foo.media.com" allow="media-playback-while-not-rendered 'none'"></iframe>
+  </body>
+</html>
+```
+
+In the past, the ["execution-while-not-rendered" and "execution-while-out-of-viewport"](https://wicg.github.io/page-lifecycle/#feature-policies) permission policies have been proposed as additions to the Page Lifecycle draft specification. However, these policies freeze all iframe JavaScript execution when not rendered, which is not desirable for the featured use case. Moreover, this proposal has not been adopted or standardized.
+
+## Interoperability with other Web API's
+
+Given that there exists many ways for a website to render audio in the broader Web Platform, this proposal has points of contact with many API's. To be more specific, there are two scenarios where this interaction might happen:
```

Comment:
Nit: "Web Platform" should not be capitalized since it's not a proper name. See https://www.w3.org/standards/ for an example of where it's not capitalized.